implementation/src/main/java/io/smallrye/config/ConfigValueConfigSourceInterceptor.java

```diff
@@ -0,0 +1,42 @@
+package io.smallrye.config;
+
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.regex.Pattern;
+
+import javax.annotation.Priority;
+
+import org.jboss.logging.Logger;
+
+@Priority(100)
+public class ConfigValueConfigSourceInterceptor implements ConfigSourceInterceptor {
```

Comment:
This interceptor is a logging interceptor and nothing more, so the name should reflect that fact.